app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -31,6 +31,8 @@ class ManageIQ::Providers::Openstack::CloudManager < EmsCloud
 
   include ManageIQ::Providers::Openstack::ManagerMixin
 
+  validates :authentications, :presence => {:message => 'You must enter credentials.'}
```

Comment:
Seems to work well in `cloud_manager`, but then don't we also need it in `infra_manager`.
I'd prefer `base_manager`.
Also it seems Foreman already requires credentials :)